drivers/tripplite-hid.c

```diff
@@ -80,6 +80,7 @@ static usb_device_id_t tripplite_usb_device_table[] = {
 	{ USB_DEVICE(TRIPPLITE_VENDORID, 0x1003), battery_scale_0dot1 },
 	/* e.g. TrippLite AVR750U */
 	{ USB_DEVICE(TRIPPLITE_VENDORID, 0x1007), battery_scale_0dot1 },
+	{ USB_DEVICE(TRIPPLITE_VENDORID, 0x3024), smart1500lcdt_scale },
```

Comment:
Thanks! Looking at precedent in this table, the ID's are alphabetically sorted which helps maintaining it without conflicts. Can you please move the line lower, probably adding a comment with the device model again (maybe adjusted for difference from existing AVR750U entry... "newer model"? revision/years known?)